packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-tile.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+  Component,
+  Prop,
+  h,
+  Element,
+  Host,
+  State,
+  Fragment,
+} from '@stencil/core';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+  OdsThemeTypographySize,
+} from '@ovhcloud/ods-theming';
+import { HTMLStencilElement, Watch } from '@stencil/core/internal';
+import { Locale, defaultLocale, formatDate } from '@ovhcloud/msc-utils';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { getTranslations } from './translations';
+import {
+  ServiceDetails,
+  ServiceInfos,
+  Translations,
+} from './msc-billing.types';
+
+export interface IMscBillingTile {
+  servicePath: string;
+  locale?: Locale;
+  commitmentDataTracking?: string;
+  changeOwnerDataTracking?: string;
+  updateOwnerDataTracking?: string;
+  subscriptionManagementDataTracking?: string;
+  renewLinkDataTracking?: string;
+  cancelResiliationDataTracking?: string;
+  manageRenewDataTracking?: string;
+  resiliateDataTracking?: string;
+  anticipateRenewDataTracking?: string;
+}
+
+@Component({
+  tag: 'msc-billing-tile',
+  styleUrl: 'msc-billing-tile.scss',
+  shadow: true,
+})
+export class MscBillingTile implements IMscBillingTile {
+  @Element() host!: HTMLStencilElement;
+
+  @Prop() public locale = defaultLocale;
+
+  @Prop() public servicePath: string;
+
+  @Prop() public commitmentDataTracking?: string = '';
+
+  @Prop() public changeOwnerDataTracking?: string;
+
+  @Prop() public updateOwnerDataTracking?: string;
+
+  @Prop() public subscriptionManagementDataTracking?: string;
+
+  @Prop() public renewLinkDataTracking?: string;
+
+  @Prop() public cancelResiliationDataTracking?: string;
+
+  @Prop() public manageRenewDataTracking?: string;
+
+  @Prop() public resiliateDataTracking?: string;
+
+  @Prop() public anticipateRenewDataTracking?: string;
+
+  @Prop() public changeOfferDataTracking?: string;
+
+  @State() localeStrings?: Translations;
+
+  @State() serviceInfos?: ServiceInfos;
+
+  @State() serviceDetails?: ServiceDetails;
+
+  @Watch('locale')
+  async updateTranslations() {
+    this.localeStrings = await getTranslations(this.locale);
+  }
+
+  async componentWillLoad() {
+    this.updateTranslations();
+
+    apiClient.v6.get(`${this.servicePath}/serviceInfos`).then((response) => {
+      this.serviceInfos = response.data;
+      apiClient.v6
+        .get(`/services/${this.serviceInfos?.serviceId}`)
+        .then((responseDetails) => {
+          this.serviceDetails = responseDetails.data;
+        });
+    });
+  }
+
+  private getServiceName() {
+    return this.servicePath.substring(this.servicePath.lastIndexOf('/') + 1);
+  }
```

Comment:
**nitpick:** Current implementation assumes that `servicePath` always ends with `serviceName`. But this hugely depends on the API design. Better to accept `serviceName` as an input to decouple from API design.

Reply:
That's true we couple the component to the API design to simplify its usage.

But anyways, if the API changes, we have to update the way we call it in the component (upgrade the package to call the API or change from v6 to v2 for example) and, when this happens, we will add the necessary parameters if needed.

I don't think this is useful right now to decouple from the API design since we are coupled to the API design in other ways.